lib/system/excpt.nim

```diff
@@ -320,6 +337,10 @@ proc raiseExceptionAux(e: ref Exception) =
       quitOrDebug()
     else:
       pushCurrentException(e)
+      raise_counter.inc
+      if raise_counter == 0:
+        raise_counter.inc # skip zero at overflow
```

Comment:
seriously? You raise the same exception more than 4 billion times? Doesn't hell freeze over before that happens in practice?